src/pages/LoginPage/components/LoginForm/LoginForm.tsx

```diff
@@ -3,10 +3,31 @@ import { Form, Formik, FormikProps } from "formik";
 import styles from "./LoginForm.module.scss";
 import { PasswordField, TextField } from "@/src/components/Fields";
 import { UIbutton } from "@/src/components/Buttons";
-import { ILoginPage } from "./LoginForm.types";
+import { ILoginPage, SetFieldError } from "./LoginForm.type";
+
 import validationSchema from "./validationSchema";
+import LoginApi from "./LoginApi";
+import { links } from "@/src/utils/links/links.types";
+import { useNavigate } from "react-router-dom";
 
 const LoginPageForm: FC = () => {
+  const navigate = useNavigate();
+
+  const onHandleSubmit = async (
+    values: ILoginPage,
+    setFieldError: SetFieldError
+  ) => {
+    const { status, data } = await LoginApi(values);
+
+    if (status === 200) {
+      localStorage.setItem("token", data.token);
+      navigate(links.FEED);
+    }
+    if (status === 422) {
+      setFieldError("email", " ");
+      setFieldError("password", "wrong email or password");
```

Comment:
if you need to set an error for 2 fields, as in your case, you can use formik setErrors method, which accepts as parameters an object with fields name as key => setErrors: (fields: { [field: string]: string }) => void, instead of the setFieldError method. but this is just a recommendation